src/UIComponents/Predict.jsx

```diff
@@ -47,7 +49,10 @@ class Predict extends Component {
                 return (
                   <div key={index}>
                     <label>
-                      {feature}: &nbsp;
+                      {feature} (min:{" "}
+                      {+this.props.rangesByColumn[feature].min.toFixed(2)}, max:{" "}
+                      {+this.props.rangesByColumn[feature].max.toFixed(2)}
```

Comment:
Could we hoist these long expressions into a couple local variables in the `render` function, and then use string interpolation to make this expression more compact?  

Also are the `+`s necessary?

Reply:
Good point! If I put the variables in the render function, it loses the context of feature from selectedNumericalFeatures.map((feature, index). Do we want to do this or is the change I just pushed okay?

Reply:
Oh I see.  There are a few ways to factor this, such as having another React component, or a sub-render function, or a new data structure entirely (such as we do with `getCrossTabData`) but I think this is okay as-is.

Reply:
Those are good things for me to think about going forward. Thank you!